packages/bot-web-ui/src/components/dashboard/joyride-config.tsx

```diff
@@ -31,77 +46,228 @@ const joyride_config: TJoyrideConfig = {
     disableBeacon: true,
 };
 
-export const DBOT_ONBOARDING = [
+export const BOT_BUILDER_TOUR = [
     {
         target: 'body',
-        content: (
+        content: [
+            <Step
+                key='Lets build a bot'
+                label={localize("Let's build a bot")}
+                content={[
+                    {
```

Comment:
we don't need to make array of objects and put only contentData inside, instead we can simply make array of strings.

Reply:
alright done.